Gems/AtomLyIntegration/CommonFeatures/Code/Include/AtomLyIntegration/CommonFeatures/Material/MaterialComponentBus.h

```diff
@@ -33,6 +33,16 @@ namespace AZ
             virtual const MaterialAssignmentMap& GetMaterialOverrides() const = 0;
             //! Clear all material overrides
             virtual void ClearAllMaterialOverrides() = 0;
+            //! Clear non-lod material override
+            virtual void ClearModelMaterialOverrides() = 0;
+            //! Clear lod material override
+            virtual void ClearLodMaterialOverrides() = 0;
+            //! Clear residual materials that don't correspond to the associated model
+            virtual void ClearIncompatibleMaterialOverrides() = 0;
+            //! Clear materials that reference missing assets
+            virtual void ClearInvalidMaterialOverrides() = 0;
+            //! Repair materials that reference missing assets by assigning the default asset
+            virtual void RepairInvalidMaterialOverrides() = 0;
```

Comment:
I considered putting some of these functions on the editor component. I think they have value for scripting or if changing models regularly.  Everything in the editor component, related tools, and some of the tests goes through this bus instead of modifying the configuration directly.